chia/full_node/subscriptions.py

```diff
@@ -48,22 +48,16 @@ def has_subscription(self, item: bytes32) -> bool:
         return item in self._peers_for_subscription
 
     def count_subscriptions(self, peer_id: bytes32) -> int:
-        if peer_id not in self._subscriptions_for_peer:
-            return 0
-
-        return len(self._subscriptions_for_peer[peer_id])
+        return len(self._subscriptions_for_peer.get(peer_id, {}))
 
     def remove_peer(self, peer_id: bytes32) -> None:
-        if peer_id not in self._subscriptions_for_peer:
-            return
-
-        for item in self._subscriptions_for_peer[peer_id]:
+        for item in self._subscriptions_for_peer.pop(peer_id, {}):
             self._peers_for_subscription[item].remove(peer_id)
 
             if len(self._peers_for_subscription[item]) == 0:
                 self._peers_for_subscription.pop(item)
 
-        self._subscriptions_for_peer.pop(peer_id)
+        self._subscriptions_for_peer.pop(peer_id, None)
```

Comment:
you don't need this line anymore, right?

Reply:
Right, good point